solidity/test/SpecAndRunRequester_test.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+require('./support/helpers.js')
+
+contract('SpecAndRunRequester', () => {
+  let Link = artifacts.require("LinkToken.sol");
```

Comment:
Because the variables in L6-9 aren't reassigned how do you feel about making these `const`?